op-program/host/subcmds/configs_cmd.go

```diff
@@ -96,3 +104,54 @@ func listChain(chainID eth.ChainID) error {
 	fmt.Println(description)
 	return nil
 }
+
+func CheckCustomChains(ctx *cli.Context) error {
+	customChainIDs, err := chainconfig.CustomChainIDs()
+	if err != nil {
+		return err
+	}
+
+	var errs []error
+	interopChains := make(map[eth.ChainID]bool)
+
+	for _, chainID := range customChainIDs {
+		cfg, err := chainconfig.RollupConfigByChainID(chainID)
+		if err != nil {
+			errs = append(errs, err)
+			continue
+		}
+		_, err = chainconfig.ChainConfigByChainID(chainID)
+		if err != nil {
+			errs = append(errs, err)
+			continue
+		}
+
+		if cfg.InteropTime != nil {
+			depset, err := chainconfig.DependencySetByChainID(chainID)
+			if err != nil {
+				errs = append(errs, err)
+				continue
+			}
+			for _, dep := range depset.Chains() {
+				interopChains[dep] = true
+			}
+		}
+		if err := listChain(chainID); err != nil {
+			return err
+		}
+	}
+
+	for chainID := range interopChains {
+		if !slices.Contains(customChainIDs, chainID) {
+			err := listChain(chainID)
+			if err != nil {
+				errs = append(errs, fmt.Errorf("chain in depset not found in superchain-registry: %w", err))
+			}
+		}
+	}
+
+	if len(errs) > 0 {
+		return fmt.Errorf("errors checking custom chains: %v", errs)
```

Comment:
I think you could do:
```suggestion
		return fmt.Errorf("errors checking custom chains: %w", errors.Join(errs))
```

Not sure it makes much difference given the typing of errors don't matter much but seems low lift.